File: Service_eHospital_DongNai_A/Implement/BenhAnTongQuat_GiayChungSinhService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class BenhAnTongQuat_GiayChungSinhService : BaseService<BenhAnTongQuat_GiayChungSinh, IBenhAnTongQuat_GiayChungSinhRepository>
    , IBenhAnTongQuat_GiayChungSinhService
    {
    private readonly IBenhAnTongQuat_GiayChungSinhRepository _BenhAnTongQuat_GiayChungSinhRepository;
    public BenhAnTongQuat_GiayChungSinhService(IBenhAnTongQuat_GiayChungSinhRepository BenhAnTongQuat_GiayChungSinhRepository) : base(BenhAnTongQuat_GiayChungSinhRepository)
    {
    _BenhAnTongQuat_GiayChungSinhRepository = BenhAnTongQuat_GiayChungSinhRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/BenhAnTongQuat_GiayChungSinh.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { BenhAnTongQuat_GiayChungSinh } from './BenhAnTongQuat_GiayChungSinh.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class BenhAnTongQuat_GiayChungSinhService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'BenhAnTongQuat_ChiTiet_Id', 'BenhAnTongQuat_Id', 'Field_1', 'Field_2', 'Field_3', 'Field_4', 'Field_5', 'Field_6', 'Field_7', 'Field_8', 'Field_9', 'Field_10', 'Field_11', 'Field_12', 'Field_13', 'Field_14', 'Field_15', 'Field_16', 'Field_17', 'Field_18', 'Field_19', 'Field_20', 'Field_21', 'Field_22', 'Field_23', 'Field_24', 'Field_25', 'Field_26', 'Field_27', 'Field_28', 'Field_29'];

    List: BenhAnTongQuat_GiayChungSinh[] | undefined;
    ListFilter: BenhAnTongQuat_GiayChungSinh[] | undefined;
    FormData!: BenhAnTongQuat_GiayChungSinh;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "BenhAnTongQuat_GiayChungSinh";
    }
}

